Tests/RouterSpec/routerSpec.swift

```diff
@@ -13,12 +13,17 @@ class RouterSpec: QuickSpec {
         describe("#Router") {
             var router: Router!
             var routesTable: RoutesTable!
+            var responseGenerator: ResponseGenerator!
+            var dataStorage: DataStorage!
             var nullAction: NullAction!
 
             beforeEach {
-                nullAction = NullAction()
                 routesTable = RoutesTable()
-                router = Router(routesTable: routesTable)
+                dataStorage = DataStorage()
+                dataStorage.addValues(key: "data", value: "fatcat")
```

Comment:
`addValues(key: __, value: __)` will work for this project. In other projects, the data might be formatted in a different way. Just food for thought